pyrevitlib/pyrevit/forms/__init__.py

```diff
@@ -20,10 +20,10 @@
 
 from pyrevit import HOST_APP, EXEC_PARAMS, DOCS, BIN_DIR
 from pyrevit import PyRevitCPythonNotSupported, PyRevitException
-from pyrevit.compat import PY3, IRONPY340
+from pyrevit.compat import IRONPY
 from pyrevit.compat import safe_strtype
```

Comment:
Solve the merge conflict by adding the `get_elementid_value_func` import

Reply:
I've solved the conflict